lib/lint/command.js

```diff
@@ -20,6 +20,7 @@ module.exports = clap.create('lint', '[fileOrPreset]')
   .option('--no-color', 'Suppress color output')
   .option('--silent', 'No any output')
 
+  .option('--warn-unused-files <base>', 'Warn about unused files in specified path. Do not use with --js-cut-dev. It might cause incorrect result')
```

Comment:
```js
.option('--warn-unused-files <dir>', 'Warn about unused files for specified path. Avoid using with --js-cut-dev since it might cause to incorrect results')
```